cg/services/orders/store_order_services/store_pacbio_order_service.py

```diff
@@ -19,95 +23,96 @@ def __init__(self, status_db: Store, lims_service: OrderLimsService):
         self.status_db = status_db
         self.lims = lims_service
 
-    def store_order(self, order: OrderIn) -> dict:
-        """Submit a batch of samples for PacBio Long Read delivery."""
-
-        project_data, lims_map = self.lims.process_lims(order=order, new_samples=order.samples)
-        status_data: dict = self.order_to_status(order)
-        self._fill_in_sample_ids(samples=status_data["samples"], lims_map=lims_map)
-        new_samples = self._store_samples_in_statusdb(
-            customer_id=status_data["customer"],
-            order=status_data["order"],
-            ordered=project_data["date"],
-            ticket_id=order.ticket,
-            samples=status_data["samples"],
+    def store_order(self, order: PacbioOrder) -> dict:
+        """Store the order in the statusDB and LIMS, return the database samples and LIMS info."""
+        project_data, lims_map = self.lims.process_lims(
+            samples=order.samples,
+            ticket=order._generated_ticket_id,
+            order_name=order.name,
+            workflow=Workflow.RAW_DATA,
+            customer=order.customer,
+            delivery_type=DataDelivery(order.delivery_type),
         )
+        self._fill_in_sample_ids(samples=order.samples, lims_map=lims_map)
+        new_samples = self.store_order_data_in_status_db(order=order)
         return {"project": project_data, "records": new_samples}
 
-    @staticmethod
-    def order_to_status(order: OrderIn) -> dict:
-        """Convert order input to status for PacBio-only orders."""
-        status_data = {
-            "customer": order.customer,
-            "order": order.name,
-            "samples": [
-                {
-                    "application": sample.application,
-                    "comment": sample.comment,
-                    "data_analysis": sample.data_analysis,
-                    "data_delivery": sample.data_delivery,
-                    "name": sample.name,
-                    "priority": sample.priority,
-                    "sex": sample.sex,
-                    "tumour": sample.tumour,
-                    "volume": sample.volume,
-                    "subject_id": sample.subject_id,
-                }
-                for sample in order.samples
-            ],
-        }
-        return status_data
-
-    def _store_samples_in_statusdb(
-        self, customer_id: str, order: str, ordered: datetime, ticket_id: str, samples: list[dict]
-    ) -> list[Sample]:
-        """Store PacBio samples and cases in the StatusDB."""
-        customer: Customer = self.status_db.get_customer_by_internal_id(
-            customer_internal_id=customer_id
-        )
-        status_db_order = Order(
-            customer=customer,
-            order_date=datetime.now(),
-            ticket_id=int(ticket_id),
-        )
+    def store_order_data_in_status_db(self, order: PacbioOrder) -> list[Sample]:
```

Comment:
maybe this should be a hidden method (`_`prefix)

Reply:
Maybe. I don't think it bothers me too much though